src/StatementFor.cpp

```diff
@@ -133,10 +133,11 @@ static unsigned int
 make_random_array_control(unsigned int bound, int &init, int &limit, int &incr, eBinaryOps &test_op, eAssignOps &incr_op, bool is_signed)
 {
 	// choose either increment or decrement
+    int oob_flipcoin = pure_rnd_flipcoin(CGOptions::array_oob_prob());
```

Comment:
We should keep track how many times the coin is head vs tail, that is, an OOB does happen, and print the number of occurrences via Bookkeeper.

Reply:
Hi, can you please point me to some example Bookkeeper code that I can follow? Thanks!